rmgpy/data/kinetics/family.py

```diff
@@ -924,6 +924,15 @@ def save_training_reactions(self, reactions, reference=None, reference_type='',
 
         # Add new reactions to training depository
         for i, reaction in enumerate(reactions):
+            # Convert coverage dependence label to species label
+            if isinstance(entry.data, (SurfaceArrhenius, StickingCoefficient, SurfaceArrheniusBEP, StickingCoefficientBEP)):
+                if reaction.kinetics.coverage_dependence:
+                    coverage_dependence = {}
+                    reaction.kinetics.coverage_dependence = coverage_dependence
```

Comment:
doesn't this line wipe `reaction.kinetics.coverage_dependence`, replacing it with the empty dictionary `coverage_dependence` that you just made? Then the next line will iterate through the empty dictionary?

Reply:
aha yes it does! typo